src/components/Notes/NoteCardView/NoteCardView.jsx

```diff
@@ -63,6 +65,37 @@ const NoteCardView = ({ note, fromArchive }) => {
     }
   };
 
+  const checkIfChecked = (label) => {
+    return tags.includes(label);
```

Comment:
nothing wrong here, but this function could've just been an individual inline function. Good use of includes method here.